tests/test_e2e_20_flow_manager.py

```diff
@@ -1007,3 +1007,119 @@ def test_080_retrieve_flows(self):
         assert len(data["00:00:00:00:00:00:00:01"]) == BASIC_FLOWS
         assert len(data["00:00:00:00:00:00:00:02"]) == BASIC_FLOWS
         assert len(data["00:00:00:00:00:00:00:03"]) == BASIC_FLOWS
+
+    def test_085_install_flows_by_switch_but_404(self):
+        """Install flows_by_switch but one of the switches
+         does not exist."""
+        payload = {
+            "00:00:00:00:00:00:00:01": {
+                "flows": [{"cookie": 12297829382473034410, "priority": 20000}]},
+            "00:00:00:00:00:00:00:11": {
+                "flows": [{"cookie": 12297829382473034410, "priority": 20000},]},
+        }
+        api_url = KYTOS_API + '/flow_manager/v2/flows_by_switch'
+        response = requests.post(api_url, data=json.dumps(payload),
+                      headers={'Content-type': 'application/json'})
+        assert response.status_code == 404, response.text
+
+    def test_090_install_flows_in_switch_list_but_404(self):
+        """Through /v2/flows install flows in a list of switches
+         but one of them does not exist."""
+        payload = {
+            "switches": ["00:00:00:00:00:00:00:01", "00:00:00:00:00:00:00:11"],
+            "flows": [{"cookie": 12297829382473034410, "priority": 20000}],
+        }
+        api_url = KYTOS_API + '/flow_manager/v2/flows'
+        response = requests.post(api_url, data=json.dumps(payload),
+                      headers={'Content-type': 'application/json'})
+        assert response.status_code == 404, response.text
+ 
+    def test_095_install_delete_flows_by_switch(self):
+        """Install and delete via flows_by_switch API request."""
+        payload = {
+            "00:00:00:00:00:00:00:01": {
+                "flows": [{"cookie": 12297829382473034410, "priority": 20000}]},
+            "00:00:00:00:00:00:00:02": {
+                "flows": [
+                    {"cookie": 12297829382473034410, "priority": 20000},
+                    {"cookie": 12297829382473034410, "priority": 2000}
+                ]},
+            "00:00:00:00:00:00:00:03": {
+                "flows": [
+                    {"cookie": 12297829382473034410, "priority": 20000},
+                    {"cookie": 12297829382473034410, "priority": 2000},
+                    {"cookie": 12297829382473034410, "priority": 200}
+                ]},
+        }
+        api_url = KYTOS_API + '/flow_manager/v2/flows_by_switch'
+        response = requests.post(api_url, data=json.dumps(payload),
+                      headers={'Content-type': 'application/json'})
+        assert response.status_code == 202, response.text
+
```

Comment:
Can you augment here to also make a request to `GET /flow_manager/v2/stored_flows` and confirm that they expected flows states are either "pending" or "installed". 

You're already verifying in the OvS switches, but prior to that since we don't have too much cov for this endpoint, it's good to have this intermediary list stored_flows too.

Reply:
Now digging into stored_flows and checking in the last added flows.